### PR TITLE
Feature/tinymce5update 3 releases 5.1.6 to 5.1

### PR DESCRIPTION
**Pull request by lorenzo-pomili**
Related Ticket: 

Description of Changes, it covers:

- Release TinyMCE 5.1.6
- Release TinyMCE 5.1.5
- Release TinyMCE 5.1.4
- Release TinyMCE 5.1

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: modules/ROOT/partials/release-notes/51new-features.adoc [hunk 1]
```diff
@@ -6,30 +6,32 @@ The following new features and enhancements were added for the {productname} 5.1
...
 ==== Enabling the mobile editor
 
 To ensure the mobile editor functions as intended, add the following `meta` tag to the `head` of pages using {productname}.
 
-```html+++<meta name="viewport" content="width=device-width, initial-scale=1">++++++</meta>+++
-
-```
+[source, html]
+----
+<meta name="viewport" content="width=device-width, initial-scale=1"></meta>
```
Comment (lnewson) [inline modules/ROOT/partials/release-notes/51new-features.adoc:31]:
```suggestion
<meta name="viewport" content="width=device-width, initial-scale=1">
```

**Review by lnewson — APPROVED** [review]

File: modules/ROOT/partials/misc/releasenotes_for_stable.adoc [hunk 2]
```diff
@@ -1 +1 @@
-NOTE: This is the Tiny Cloud and TinyMCE Enterprise release notes. For information on the latest community version of TinyMCE, see: [TinyMCE Changelog](https://www.tiny.cloud/docs/changelog/).
+NOTE: This is the Tiny Cloud and TinyMCE Enterprise release notes. For information on the latest community version of TinyMCE, see: link:https://www.tiny.cloud/docs/changelog/[TinyMCE Changelog].
```
[the remark this answers is not in the document]
Comment (lorenzo-pomili) [reply modules/ROOT/partials/misc/releasenotes_for_stable.adoc]: done

File: modules/ROOT/partials/release-notes/51new-features.adoc [hunk 3]
```diff
@@ -57,71 +59,78 @@ tinymce.init({
 NOTE: No enhancements have been made to the legacy mobile theme. All the changes described in these release notes relate to the `silver` theme on mobile devices.
 
 For information on:
```
[the remark this answers is not in the document]
Comment (lorenzo-pomili) [reply modules/ROOT/partials/release-notes/51new-features.adoc:61]: done

File: modules/ROOT/partials/release-notes/51new-features.adoc [hunk 4]
```diff
@@ -57,71 +59,78 @@ tinymce.init({
...
 For example:
 
-image::{baseurl}/images/table_cell_touch_selector_handles.png[Touch selector handles for selecting multiple table cells,380]
+image::table_cell_touch_selector_handles.png[Touch selector handles for selecting multiple table cells,380]
 
+[[touch-friendlysplitbuttons]]
 ==== Touch-friendly split buttons
 
-The styling on link:{baseurl}/ui-components/typesoftoolbarbuttons/#splitbutton[split buttons] has been updated to include more padding so they are easier to interact with on touch devices.
+The styling on xref:ui-components/typesoftoolbarbuttons.adoc#splitbutton[split buttons] has been updated to include more padding so they are easier to interact with on touch devices.
```
[the remark this answers is not in the document]
Comment (lorenzo-pomili) [reply modules/ROOT/partials/release-notes/51new-features.adoc:113]: done

File: modules/ROOT/partials/release-notes/51new-features.adoc (hunk 1 above)
Comment (lorenzo-pomili) [reply modules/ROOT/partials/release-notes/51new-features.adoc]: done

**Review by metricjs — APPROVED** [review]

**Pull request merged**